Gemfile

```
@@ -11,7 +11,7 @@ group :development do
   # > 2.1.0 causes slowdowns https://github.com/sass/sassc-ruby/issues/189
   gem 'sassc', '2.1.0'
   gem 'sass-embedded', '1.83.4'
-  gem 'rubocop', '1.70.0'
+  gem 'rubocop', '1.71.0'
```

rubocop (1.69.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `rubocop` should appear before `sass-embedded`.

(Bundler/OrderedGems)